src/Avalonia.Themes.Fluent/Controls/MenuItem.xaml

```diff
@@ -112,6 +112,7 @@
           <Popup Name="PART_Popup"
                  WindowManagerAddShadowHint="False"
                  PlacementMode="Right"
+                 OverlayInputPassThroughElement="{Binding $parent[MenuItem]}"
```

Comment:
Probably it isn't required anymore after https://github.com/AvaloniaUI/Avalonia/pull/5322 was merged.
I need to test it later.

Reply:
Checked, it's still required.